draft-ietf-quic-transport.md

```diff
@@ -2314,33 +2312,31 @@ Therefore, a migrating endpoint can send to its peer knowing that the peer is
 willing to receive at the peer's current address. Thus an endpoint can migrate
 to a new local address without first validating the peer's address.
 
+To establish reachability on the new path, an endpoint initiates path	
+validation ({{migrate-validate}}) on the new path.  An endpoint MAY defer path	
+validation until after a peer sends the next non-probing frame to its new	
+address.
+
 When migrating, the new path might not support the endpoint's current sending
 rate. Therefore, the endpoint resets its congestion controller and RTT estimate,
 as described in {{migration-cc}}.
 
 The new path might not have the same ECN capability. Therefore, the endpoint
 verifies ECN capability as described in {{ecn}}.
 
-To establish reachability on the new path, an endpoint initiates path
-validation ({{migrate-validate}}) on the new path.  An endpoint MAY defer path
-validation until after a peer sends the next non-probing frame to its new
-address.
-
-Path validation is necessary to verify reachability of a peer on a new network
-path.  Acknowledgments cannot be used for path validation as they contain
-insufficient entropy and might be spoofed.  No method is provided to establish
-return reachability, as endpoints independently determine reachability on each
-direction of a path.
```

Comment:
I think that the two latter sentences are still worth keeping.  However, I would be happier if it were moved higher though.

Reply:
SG, moved to the third paragraph of "Path Validation {#migrate-validate}", PTAL